src/monitors/commandHandler.js

```diff
@@ -43,6 +44,7 @@ module.exports = class extends Monitor {
 
 	getPrefix(msg) {
 		if (this.prefixMention.test(msg.content)) return { length: this.nick.test(msg.content) ? this.prefixMentionLength + 1 : this.prefixMentionLength, regex: this.prefixMention };
+		if (this.regexPrefix.test(msg.content)) return { length: msg.content.match(this.regexPrefix) ? msg.content.match(this.regexPrefix)[0].length : null, regex: this.regexPrefix };
```

Comment:
null is never a valid value for length

Reply:
also, if regexPrefix is not defined, you will get error cannot read property test of null...

Reply:
Latest commit should fix that